.prettierignore

```diff
@@ -0,0 +1,10 @@
+# Ignore directories:
+packages/build
+packages/teleterm/src/services/tshd/v1/**
+packages/teleterm/src/sharedProcess/**
```

Comment:
I'm pretty sure this should be `packages/teleterm/src/sharedProcess/api/protogen/**` instead, as `protogen` is the actual directory that holds automatically generated files.